### DISCUSSION TITLE
How we can remove particular label from a link

### DISCUSSION THREAD

**Discussion by vyshak-pr** [clientIO/joint#1982] [Q&A] [answered]
![labeld](https://user-images.githubusercontent.com/120721238/211834637-10cdc28f-6a23-4cd8-bac7-3e5119137760.png)

How we can remove a particular label from a link

**Comment by kumilingus** [comment, marked as answer]
There is an API do delete a label at given index.

```js
link.removeLabel(1);
```
For more info see [docs](https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#dia.Link.prototype.removeLabel).

This enables you, for instance, to listen to the double-click event and remove the label as follows:

```js
paper.on('link:pointerdblclick', (linkView, evt) => {
    const labelIdx = linkView.findAttribute('label-idx', evt.target);
    if (labelIdx !== null) {
        linkView.model.removeLabel(parseInt(labelIdx));
    }
});
```

Here's a working [example](https://jsfiddle.net/kumilingus/rdo2g6z9/).

**Reply by vyshak-pr** [reply]
Thanks @kumilingus
